jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ProjectFileBrowseListener.kt

```diff
@@ -8,14 +8,17 @@ import com.intellij.openapi.project.Project
 import com.intellij.openapi.project.guessProjectDir
 import com.intellij.openapi.ui.TextBrowseFolderListener
 import com.intellij.openapi.vfs.VirtualFile
+import software.aws.toolkits.core.utils.tryOrNull
 
+/** Similar to [com.intellij.openapi.ui.TextBrowseFolderListener], but tries to set the initial directory to the assumed project root **/
 open class ProjectFileBrowseListener @JvmOverloads constructor(
     project: Project,
     chooserDescriptor: FileChooserDescriptor,
     private val onChosen: ((VirtualFile) -> Unit)? = null
 ) : TextBrowseFolderListener(chooserDescriptor, project) {
     override fun getInitialFile(): VirtualFile? {
-        val text = componentText
+        // we use this listener on ComboboxWithBrowseButton, but it will never return valid text since it's not a JTextField
+        val text = tryOrNull { componentText } ?: return null
```

Comment:
Correct way to fix this is make the text accessor configurable and pass STRING_COMBOBOX_WHOLE_TEXT

Reply:
Can probably backport this: 

https://github.com/JetBrains/intellij-community/blob/f580ac1d225106a01e2b72e702ca26195ae7ee60/platform/platform-impl/src/com/intellij/ui/components/components.kt#L241-L281